.github/workflows/python-package.yml

```diff
@@ -1,6 +1,10 @@
 name: Python package
 
-on: [pull_request]
+on: 
+    pull_request:
+    push:
+        tags:
+            - v*
```

Comment:
maybe easier just to point to master for pushes?

Reply:
Well, I don't want the release steps to trigger for any old push against master- just for tags.